src/Cli.Tests/TestHelper.cs

```diff
@@ -1065,6 +1065,104 @@ public static Process ExecuteDabCommand(string command, string flags)
         }
       }";
 
+        public const string COMPLETE_CONFIG_WITH_RELATIONSHIPS_NON_WORKING_CONN_STRING = @"
+        {
+  ""$schema"": ""https://github.com/Azure/data-api-builder/releases/download/vmajor.minor.patch/dab.draft.schema.json"",
```

Comment:
are there other const strings in this test code that this const can build up on?

Reply:
we'd have to modify them because the entities are not formed propery, with the exception of the datasource section